data/sql/grid_transformation_custom.sql

```diff
@@ -84,3 +84,38 @@ INSERT INTO "grid_transformation" VALUES(
     NULL,
     'EPSG','8666','Geoid (height correction) model file','Icegeoid_ISN2016.gtx',
     NULL,NULL,NULL,NULL,NULL,NULL,NULL,0);
+
+-- Norway
+
+ INSERT INTO "grid_transformation" VALUES(
+    'PROJ','EPSG_4937_TO_EPSG_5776','ETRS89 to NN1954 height',
+    NULL,NULL,
+    'EPSG','9665','Geographic3D to GravityRelatedHeight (tif)',
```

Comment:
I'd prefer we stick with the official name for EPSG:9665 'Geographic3D to GravityRelatedHeight (gtx)'  even if it is not super appropriate here, but making a new name while still using EPSG:9665 could be confusing. I guess EPSG will end up attributing a proper code for TIFF in the future (especially if you submit those grids)